cedar-policy-validator/src/lib.rs

```diff
@@ -319,4 +319,71 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
+    fn test_action_namespace_inference_multi() {
```

Comment:
I'd rather move this test to `schema.rs`, and then update it to specifically assert that schema that's built contains an action with the expected `memberOf`. Also the schema can use `json!` macro.